source/deprecations/ember-data/v2.x.html.md

```diff
@@ -110,3 +110,85 @@ Note: The existing behavior of
 [`DS.DateTransform`](http://emberjs.com/api/data/classes/DS.DateTransform.html)
 was preserved. It is the direct usage of `Ember.Date.parse` and overriding of
 `Date.parse` that has been deprecated.
+
+#### Store.queryRecord Array Response with RESTSerializer
+
+###### until: 3.0.0
+###### id: ds.serializer.rest.queryRecord-array-response
+
+When using
+[`DS.RESTSerializer`](http://emberjs.com/api/data/classes/DS.RESTSerializer.html)
+with previous versions of Ember Data,
+[`store.queryRecord`](http://emberjs.com/api/data/classes/DS.Store.html#method_queryRecord)
+provided support for normalizing payloads containing an array of primary data.
+This behavior has been deprecated.
```

Comment:
I think it would be good to point out why this has been deprecated: because it basically not different to a `store.query` with getting the first object of the response.